addons/website/static/src/builder/plugins/options/card_image_option_plugin.js

```diff
@@ -96,12 +108,42 @@ export class AddCoverImageAction extends BuilderAction {
         editingElement.classList.add("o_card_img_top");
     }
 }
-export class AlignCoverImageAction extends BuilderAction {
-    static id = "alignCoverImage";
-    apply({ editingElement, params: { mainParam: direction } }) {
-        const imgWrapper = editingElement.querySelector(".o_card_img_wrapper");
-        imgWrapper.classList.toggle("o_card_img_adjust_v", direction === "vertical");
-        imgWrapper.classList.toggle("o_card_img_adjust_h", direction === "horizontal");
+export class CoverImagePositionOverlayAction extends BuilderAction {
+    static id = "coverImagePositionOverlay";
+    static dependencies = ["overlayButtons", "history", "cardImageOption"];
+    setup() {
+        this.withLoadingEffect = false;
+    }
+    async load({ editingElement }) {
+        const imageEl = editingElement.querySelector(".o_card_img");
```

Comment:
Do we want to have the same than in `BackgroundPositionOverlayAction` (something with `await loadImage`)?